.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: macOS-11
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
This change is orthogonal but I noticed warnings in [this build](https://github.com/dfed/CacheAdvance/actions/runs/3230063689) (search for `warnings` on that page) and decide to fix them while I was here.